developer.md

```diff
@@ -0,0 +1,34 @@
+# Python Client Library
+
+### Documentation
+
+   [stable](http://genestack-client.readthedocs.io/en/stable/)
+
+### Installation
+
+   - install library
+     - `pip install genestack-client --upgrade`
+   - setup default user
+     - `genestack-user-setup init`
+
```

Comment:
I disagree with moving all these into another file.  **Documentation** and **Installation** sections belong to README.